src/content/en/2022/accessibility.md

```diff
@@ -1,22 +1,791 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Accessibility
-#TODO - Review and update chapter description
-description: Accessibility chapter of the 2022 Web Almanac covering ease of reading, media, ease of navigation, and compatibility with assistive technologies.
-authors: [scottdavis99, SaptakS]
-reviewers: [shantsis, alextait1, ericwbailey]
-analysts: [scottdavis99, thibaudcolas]
+description: Accessibility chapter of the 2022 Web Almanac covering ease of reading, navigation, forms, media, ARIA, and accessibility apps.
+authors: [SaptakS, scottdavis99, thibaudcolas]
+reviewers: [shantsis]
+analysts: [thibaudcolas]
 editors: [dereknahman]
```

Comment:
@SaptakS can you confirm this list is correct and, within each category, contributors are listed in descending order of contribution?

Reply:
About the order, I would change authors order to [SaptakS, thibaudcolas, scottdavis99]. Rest looks good to me.

Reply:
```suggestion
authors: [SaptakS, thibaudcolas, scottdavis99]
reviewers: [shantsis]
analysts: [thibaudcolas]
editors: [dereknahman]
```